manifests/overlays/dev/mock-server.yaml

```diff
@@ -0,0 +1,138 @@
+# This YAML adds a mock server of gpu metrics for gpu testing
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nvidia-dcgm-exporter
+  labels:
+    app: nvidia-dcgm-exporter
+spec:
+  replicas: 1
+  selector:
+    type: RollingUpdate
+    matchLabels:
+      app: nvidia-dcgm-exporter
+  template:
+    metadata:
+      labels:
+        app: nvidia-dcgm-exporter
+    spec:
+      containers:
+        - name: metrics
+          image: registry.redhat.io/rhel8/nodejs-14-minimal:latest
+          imagePullPolicy: Always
+          workingDir: /usr/tmp
+          command:
+            - /bin/bash
+            - -c
+            - |
+              #/bin/bash
+              cat <<EOF>> package.json
+              {
+                  "type": "module"
+              }
+              EOF
+              cat <<EOF > mock-server.js
+              import client from 'prom-client';
+              import http from 'http';
+              import url from 'url';
+
+              const registry = new client.Registry
+
+              const mockGPU = new client.Gauge(
+                  {
+                      name: "DCGM_FI_PROF_GR_ENGINE_ACTIVE",
+                      help: "Mock gpu data",
+                      labelNames: [
+                          'Hostname',
+                          'UUID',
+                          'device',
+                          'gpu',
+                          'instance',
+                          'endpoint',
+                          'namespace',
```

Comment:
These are not labels of DCGM_FI_PROF_GR_ENGINE_ACTIVE metrics, they are added by Prometheus scrapping. Remove them.